Heroes Handbook, Main File

```diff
@@ -330,6 +330,52 @@ Many races have subraces. Members of a subrace have the traits of the parent rac
 
 \pagebreakNum
 
+# Alliance Class Suggestions
+
+The following are a set of class suggestions, pulled from World of Wacraft, as the classes the Alliance races will more than likely be. It is suggested that you pick from among your race's suggested classes to keep with the thematic elements of the setting. That said, heroes are meant to break molds and be the exception to rules, so feel free to pick a class that is not typically used by your race. If you do pick outside your racial classes though, work with your Dungeon Master to create a backstory for why you are a member of this class. 
+
+You will likely notice that there are two class suggestion charts. One for Pre-Cataclysm, and one for Post-Cataclysm. This was done because the Cataclysm is a major turning point in the history of Azeroth, and the races of the Alliance began learning new abilities and sources of power, in order to fight back against the threats that assail their world. Prior to the Cataclysm, the Alliance races were not as open to sharing knowledge of and learning about new classes and tended to stay within the traditional classes of their race.
+
+<div class='wide'>
+
+## Pre-Cataclysm
+| Race | Class Suggestions |
+|:----|:-------------|
+| Human  | Mage, Paladin, Priest, Rogue, Warrior, Warlock, Death Knight |
+| Bronzebeard Dwarf  | Hunter, Paladin, Priest, Rogue, Warrior, Death Knight |
+| Wildhammer Dwarf | Hunter, Shaman, Warrior |
+| Gnome | Mage, Rogue, Warlock, Warrior, Death Knight |
+| Exodar Draenei | Hunter, Mage, Paladin, Priest, Shaman, Warrior, Death Knight |
+| Broken Draenei (Krokul) | Hunter, Mage, Rogue, Shaman, Warlock, Warrior |
+| Night Elf (Kaldorei) | Druid, Hunter, Priest, Rogue, Warrior, Death Knight |
+| High Elf (Quel'dorei) | Hunter, Mage, Paladin, Priest, Rogue, Warlock, Warrior, Death Knight |
+
+</div>
+
+<div class='wide'>
+
+## Post-Cataclysm
+| Race | Class Suggestions |
+|:----|:-------------|
+| Human | Hunter, Mage, Paladin, Priest, Rogue, Warrior, Warlock, Death Knight, Monk |
```

Comment:
I would add Demon Hunter to Humans Post-Cataclysm list, it is in the lore for demon hunters, and it widens the races that can be Demon Hunters a tiny bit